api/protocol/message_0.proto

```diff
@@ -184,6 +184,8 @@ message MessageV0 {
     optional Type type = 9 [default = VHOST_USER_SERVER];
     // Nic path may be set depending of NIC's type
     optional string path = 10;
+    // dump VM traffic in pcap file
+    optional bool packet_trace = 11;
```

Comment:
would't be interesting to set a string instead of a bool so the user can choose in which file he wants his pcap to be written ? If the user sets an empty string (but the field is set), then an automatic file could be chosen.

Reply:
I think we have to add another option for the path, because we trace all nics by default, and if i set this option to sting, we will not get option to disable packet tracing